eng/src/BuildMetalamaPatterns.csproj

```diff
@@ -7,7 +7,7 @@
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <LangVersion>latest</LangVersion>
         <Nullable>enable</Nullable>
-        <NoWarn>SA0001;CS8002</NoWarn>
+        <NoWarn>SA0001;CS8002;NU1903;NU1904</NoWarn>
```

Comment:
Comment on what these warnings mean, why they are suppressed, and when (if ever) this suppression should be removed.